src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultWindowsRazorProjectHost.cs

```diff
@@ -162,7 +162,7 @@ protected override async Task HandleProjectChangeAsync(string sliceDimensions, I
             return false;
         }
 
-        configuration = new(languageVersion, configurationItem.Key, extensions, ForceRuntimeCodeGeneration: forceRuntimeCodeGeneration);
+        configuration = new(languageVersion, configurationItem.Key, extensions, RazorLanguageFeatureFlags.Default with { ForceRuntimeCodeGeneration = forceRuntimeCodeGeneration });
```

Comment:
Consider passing `_languageServerFeatureOptions` into this method, to avoid using the `Default` flags, so any future flags are handled automatically (or see other comment about removing Default)